labs-ios-starter/Surveys/View Controllers/TopicNameViewController.swift

```diff
@@ -5,19 +5,79 @@ import UIKit
 
 class TopicNameViewController: UIViewController {
     // MARK: - Outlets -
-
     @IBOutlet var nameTextField: UITextField!
+    @IBOutlet var contextPicker: UIPickerView!
 
     // MARK: - Properties -
+    let topicController = TopicController()
+    let fetchController = FetchController()
```

Comment:
Should we be injecting these through navigation? Or would that save space at the expense of reliability when trying to pass the same instance of a controller throughout the app? Something to think about

Reply:
We should avoid instantiating more than one even though they're really just holding common methods and won't munge any stored data. You caught me being lazy again